README.md

```diff
@@ -32,7 +32,7 @@ dnf copr enable mperina/hirte-snapshot
 When done you can install relevant hirte packages using:
 
 ```bash
-dnf install hirte hirte-agent
+dnf install hirte hirte-agent hirte-ctl
```

Comment:
Isn't it `hirtectl`?

Reply:
Ah ok, was thinking in terms of the compiled binary. Thanks for the explanation!